content/docs/0.8.1/concepts.md

```diff
@@ -73,4 +87,31 @@ If the current healthy replica count is more than specified replica count, Longh
 ### Volume parameters
 
 #### Stale Replica Timeout 
-Stale Replica Timeout (`staleReplicaTimeout`) determines when would Longhorn cleanup an error replica after the replica become `ERROR`. Unit is in minutes. Default is `2880` (48 hours)
\ No newline at end of file
+Stale Replica Timeout (`staleReplicaTimeout`) determines when would Longhorn cleanup an error replica after the replica become `ERROR`. Unit is in minutes. Default is `2880` (48 hours)
+
+### Disaster Recovery Volumes
+
+A disaster recovery volume (DRV) is a special volume that stores data in a backup cluster in case the whole main cluster goes down. DRVs are used to increase the resiliency of Longhorn volumes.
+
+Because the main purpose of a DRV is to restore data from backup, this type of volume doesn't support creating/deleting/reverting snapshot, creating backups, creating persistent volumes (PVs) and persistent volume claims (PVCs).
```

Comment:
Add `before activated` in the end.

Reply:
updated

Reply:
The sentence was getting long so i did this:

> Because the main purpose of a DR volume is to restore data from backup, this type of volume doesn't support the following actions before it is activated: 
> 
> - Creating, deleting, and reverting snapshots
> - Creating backups
> - Creating persistent volumes
> - Creating persistent volume claims